src/app/login/login/login.component.css

```diff
@@ -0,0 +1,3 @@
+.marginLeft{
```

Comment:
css naming should be like .margin-left, by the way, please rename it to something logic for instance like: .login-input or any other depends on who is using this style